### PR TITLE
Removed useless errors

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- Removed useless errors that are triggered when the request failed;
- We already have global errors for this case;

### CODE REVIEW

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/HistoryView.kt [hunk 1]
```diff
@@ -254,7 +246,6 @@ class HistoryView : AbstractView<HistoryProps, HistoryViewState>(false) {
             setState {
                 isDeleteExecutionWindowOpen = false
             }
-            window.location.reload()
```
Comment (petertrr) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/HistoryView.kt:257]: Removing this line should also fix #755

**Review by petertrr — APPROVED** [review]

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt [hunk 2]
```diff
@@ -502,6 +489,9 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
                     }
                     val newGitDto = gitDto?.copy(url = urlWithTests, branch = newBranch, hash = newCommit)
                         ?: GitDto(url = urlWithTests, branch = newBranch, hash = newCommit)
+
+                    console.log(newGitDto)
```
Comment (kgevorkyan) [inline save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt:493]: Maybe add this one too into ForbiddenMethodCall in detekt?

File: save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt [hunk 3]
```diff
@@ -253,6 +254,21 @@ class ExecutionController(private val executionService: ExecutionService,
             }
     }
 
+    @GetMapping(path = ["/api/$v1/getTestRootPathByExecutionId"])
+    @Transactional
+    fun getTestRootPathByExecutionId(@RequestParam id: Long, authentication: Authentication): Mono<String> =
+        Mono.justOrEmpty(executionService.findExecution(id))
+            .switchIfEmpty() {
+                Mono.error(ResponseStatusException(HttpStatus.NO_CONTENT))
```
Comment (petertrr) [inline save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt:262]: Why `204`? Maybe, `NOT_FOUND`?

Comment (akuleshov7) [reply save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/ExecutionController.kt:262]: Yeah, good idea. My intention was to explicitly mark that we don't have content in the response

File: save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt (hunk 2 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt]: Mentioned that in https://github.com/analysis-dev/diktat/issues/954 assigned to @nulls

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt]: Both methods: `['kotlin.js.console.log']` and `['kotlin.js.console']` do not work

**Pull request merged**